packages/project/src/merge/merge.ts

```diff
@@ -18,28 +20,48 @@ type Options = {
  * target, but the UUIDs of the source
  */
 // TOOD what if a workflow is removed from the target?
-export function merge(source: Project, target: Project, options) {
+export function merge(
+  source: Project,
+  target: Project,
+  options: Options = { workflowMappings: {} }
```

Comment:
As well as mappings, we need an option for "only transfer these workflows from the source"